test/PublishCommand.js

```diff
@@ -964,49 +983,49 @@ describe("PublishCommand", () => {
         [ChildProcessUtilities, "execSync", {}, [
           { args: ["git tag"] }
         ]],
-       [PromptUtilities, "confirm", { valueCallback: true }, [
```

Comment:
How in the world did it pass otherwise? If you `npm run lint` in master right now, it makes absolutely no complaint about this weird indentation.

(the consistency apparently is trying to validate that the closing square bracket is on the same column as the opening square bracket, not the general indent, which is frustrating)